### PR TITLE
Add Scroll to Top button

### LINKED ISSUE #3185: Contact button lacks a back top top button
**Actual Behaviour**
There's no **Back to Top** button on Contact Page
**Expected Behaviour**
_Back to Top_ button is available on every page but it's not on _Contact_ page
**Steps to reproduce it**
Go to [susi.ai/contact](https://susi.ai/contact) and scroll. You'll not see any _Back to Top_ button

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by MuhammadWasif**
Fixes #3185 

**Changes:**
I've added scroll to top button in contact page

**Demo Link:** https://musing-meninsky-855bca.netlify.com/
Screenshots of the change: 
ezgif-1-97186df8a97e

### CODE REVIEW

**Comment by anishagg17** [comment]
> We have to write some common logic and placer it at the root component, just like the Chat bubble is present.

Sir, Can you please assign me that issue

**Comment by MuhammadWasif** [comment]
> Sir, Can you please assign me that issue

I'm working on this one already, man!

**Comment by MuhammadWasif** [comment]
> We have to write some common logic and placer it at the root component, just like the Chat bubble is present.

But the **Scroll to Top** button is implemented on all other pages in the same way...! 😕

**Comment by MuhammadWasif** [comment]
> @MuhammadWasif I understand. But since the code is getting redundant, we need to write better code to avoid duplicate code.

Then, what should I do? What's the other way?

**Comment by MuhammadWasif** [comment]
@akshatnitd After implementing the button as **Chat Bubble**, should I remove the **ScrollTopButton** from every page? Because if I don't it will be rendered two times on every page.

**Comment by MuhammadWasif** [comment]
@anishagg What...? I've been working on it since three days. I opened this pull request 3 days ago. And I was discussing changes with mentor. Bro, yoh duplicated the work. That's not fair.

**Comment by anishagg17** [comment]
> @anishagg17 What...? I've been working on it since three days. I opened this pull request 3 days ago. And I was discussing changes with mentor. Bro, yoh duplicated the work. That's not fair.

wait bro weren't you working on `Contact button lacks a back top top button #3185`
You never opened any issue related to this pull request 
the issue was opened and fixed by my pr which i made more than 20hrs ago . 
You  have modified your pull request after seeing mine 

And yeh 

`Bro, yoh duplicated the work` 
How can i duplicated  your work before it was published

` I've been working on it since three days.`
wait bro for past 2 days you didn't knew how to figure out the fix . But after i made  a pr you instantaneously  got the fix.

File: src/App.js [hunk 1]
```diff
@@ -261,7 +266,7 @@ class App extends Component {
             )}
             {renderAppBar}
             {renderAppBanner}
```
Comment (akshatnitd) [inline src/App.js]: Directly use ScrollToTop here, instead of adding it in the function.

Comment (MuhammadWasif) [reply src/App.js]: Done...! Please merge, now!

**Comment by MuhammadWasif** [comment]
@akshatnitd Done...Please merge!  🥺 🥺

**Pull request merged**